week1/scary/kirikaza/Scary.hs

```diff
@@ -0,0 +1,19 @@
+module Main where
+
+import Data.Char (isAsciiLower, isAsciiUpper, ord)
+
+main :: IO ()
+main = do
+  file <- getContents
+  let words = lines file
+  let scaryWords = filter isScary words
+  mapM_ putStrLn scaryWords
+
+isScary :: String -> Bool
+isScary word = foldl plusLetter 0 word == 13
+  where plusLetter sum char = sum + weigh char
```

Comment:
Generally it's better to write things with `map` if possible (not better as in “objectively better”, just people are more used to it and the code would be more readable). In this case, for instance, it'd look like `sum (map letterCode word) == 13` (with appropriate definition of `letterCode`).